src/ParaTest/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -321,15 +338,14 @@ protected function getDefects(array $defects, $type)
     /**
      * Prints progress for large test collections
      */
-    protected function printProgress()
+    protected function getProgress()
     {
-        printf(
-            ' %' . $this->numTestsWidth . 'd (%3s%%)',
+        return sprintf(
+            ' %' . $this->numTestsWidth . 'd / %' . $this->numTestsWidth . 'd (%3s%%)',
             $this->casesProcessed,
-            floor(($this->casesProcessed / $this->totalCases) * 100)
+            $this->totalCases,
```

Comment:
Also, I need to note that old version (before my fixes) even if it's not show total amount still show wrong percent, so from my point of vision showing self-corrected total amount of tests is not so bad because progress percent is self corrected too.